lesson_9/homework_09.py

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
The last comment - if your prepare PR and pushed changes - this changes are used to be final - than they are finished. In finished code it is not good to store unnecessary comments. Better not to use comments at all if they are not very important for code understanding.